core/tests/webdriverio_utils/waitFor.js

```diff
@@ -57,10 +57,10 @@ var elementToBeClickable = async function(element, errorMessage) {
     timeout: DEFAULT_WAIT_TIME_MSECS,
     timeoutMsg: errorMessage + '\n' + new Error().stack + '\n'
   });
```

Comment:
@siddharth-singh-bhadouria what are these changes about, they maybe causing the lint checks to fail. Would you get rid of this? Thanks!

Reply:
@SanjaySajuJacob do you mean the other changes in this file or changes in the highlighted lines? because i don't see any change in the highlighted lines . If there is i will take another look at it .